packages/cms/README.md

```diff
@@ -202,6 +203,58 @@ A Canon site often takes the form of DataCountry.io, and is made of **Profiles**
 
 ---
 
+## Advanced Commands
+
+For complex visualizations, it may be necessary to write custom code that causes another event to fire. There are two primary use cases here:
+
+### Opening another visualization
```

Comment:
this will more often be referred to as "interacting between visualizations" or "visualization click events changing page variables/state", we shouldn't lead with "opening another visualization" (that's not actually how they're using it in OEC)

Reply:
maybe name the section "Advanced Visualization Techniques", and then this sub-header "Modifying Page State"